packages/big-design/src/components/ButtonGroup/styled.tsx

```diff
@@ -0,0 +1,57 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { StyleableButton } from '../Button/private';
+import { FlexItem } from '../Flex';
+
+interface StyledButtonProps {
+  borderRadius?: boolean;
+}
+
+interface StyledFlexItemProps {
+  isVisible: boolean;
+}
+
+export const StyledButton = styled(StyleableButton)<StyledButtonProps>`
+  ${({ borderRadius, theme }) =>
+    borderRadius
+      ? css`
+          border-radius: ${theme.borderRadius.normal};
+        `
+      : css`
+          border-radius: ${theme.borderRadius.none};
+        `}
+
+  &:focus {
+    z-index: 1;
+  }
```

Comment:
I use `z-index: 1` for correct displaying of `box-shadow` (pls see the example below). I decide to not use the value from `theme` cause it will be not semantic for my case and I think creating a new one will be redundant.


**Without `z-index`**
<img width="346" alt="Screenshot 2021-04-22 at 10 29 58" src="https://user-images.githubusercontent.com/9980803/115674046-f00f1500-a355-11eb-97a2-b7a6445d0514.png">

**With `z-index`**
<img width="296" alt="Screenshot 2021-04-22 at 10 29 46" src="https://user-images.githubusercontent.com/9980803/115674093-fb624080-a355-11eb-9a38-2fc8ed6eaedc.png">